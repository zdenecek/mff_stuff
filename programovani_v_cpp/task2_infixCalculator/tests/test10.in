11
-3
12.3 + 3 * 2
x=20 / (10-2)
y=x
y=$x
y=$x * $x
xyz=         -(1 + 2 - ((-3) * (2)))
wdisisususus
$Y * $X
 z=3
zz=$z