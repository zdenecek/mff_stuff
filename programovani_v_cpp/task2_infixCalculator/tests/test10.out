-3
18.3
2
invalid
2
invalid
-9
invalid
4
invalid
invalid
